project/urbanity/urbanity.vcxproj

```diff
@@ -157,10 +160,13 @@ py $(ProjectDir)\..\..\scripts\copy_dlls.py $(PlatformTarget) $(Configuration)
       <PreprocessorDefinitions>_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
+      <AdditionalIncludeDirectories>C:\Development\urbanity\dependencies\piranha\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
All directories should be relative. It's done correctly later on in this project file but this path won't work on anyone else's computer.

Reply:
How do I make it relative?

Reply:
$(ProjectDir) I suppose

Reply:
Yes exactly, something like `$(ProjectDir)/../../dependencies/piranha/include/`. Also make sure that this applies to all configurations and architectures